stdlib/buffer.mli

```diff
@@ -18,7 +18,16 @@
    This module implements buffers that automatically expand
    as necessary.  It provides accumulative concatenation of strings
    in quasi-linear time (instead of quadratic time when strings are
-   concatenated pairwise).
+   concatenated pairwise). For example:
+
+{[
+            let concat_strings ss =
+              let b = Buffer.create 16 in
+                List.iter (Buffer.add_string b) ss;
+                Buffer.contents b
+
+]}
```

Comment:
Out of curiosity: is there a reason for picking a certain indentation for your code samples? I would guesstimate that the two use different indentations, and it is not particularly related to the indentation of the text block before it.

Reply:
I am just trying to copy roughly what other modules do. I will have a look at all the modules to establish a fixed indentation and add it to the pull request...